stack.yaml

```diff
@@ -2,3 +2,5 @@ resolver: nightly-2018-07-28
 packages:
 - cursor
 - cursor-gen
+nix:
+  path: [ "nixpkgs=https://github.com/NixOS/nixpkgs/archive/886871538c15681575925a990a3f1b68ed9d5477.tar.gz" ]
```

Comment:
That does nixpkgs version does not correspond to `nightly-2018-07-28`, don't add it here.
You can use `lts-12.yaml`.

Reply:
What does "correspond" mean here?
I added it because it contains the correct ghc version so that stack + nix works with this stack.yaml.